resources/metadata/fortifyExecuteScan.yaml

```diff
@@ -387,7 +387,7 @@ spec:
           - STAGES
           - STEPS
         default: ["./lib", "."]
-        deprecated: true
+        deprecationMessage: this is deprecated
```

Comment:
@OliverNocon @nevskrem do you remember why `pythonAdditionalPath` of `fortifyExecuteScan` is deprecated, so that we can write a proper deprecation message?

Reply:
I don't but maybe @nevskrem knows ...